core/app/modules/site-editor/assets/js/module.js

```diff
@@ -161,6 +161,21 @@ export default class SiteEditor {
 					video_url: 'https://www.youtube.com/embed/ACCNp9tBMQg',
 				},
 			},
-			];
+			{
+				type: 'loop',
+				icon: 'eicon-error-404', // this icon needs to be updated
```

Comment:
please update it and remove the comment 
I suggest using "TODO:  " comments instead , but this cannot go to PR

Reply:
updated